distributed_training/validator/forward.py

```diff
@@ -159,6 +160,17 @@ async def forward(self):
                     time.sleep(1)
 
                 if gradient_averaging_step.done():
+                    
+                    gathered, failed_senders, participating_peers = gradient_averaging_step.result()
+                    
+                    print("\n")
```

Comment:
Awesome work @mikkelmedm. Could we just change the print statement to be bittensor logs instead.